### PR TITLE
Add warning about schema change

### PR DESCRIPTION
**Pull request by octonato**
References #504

### CODE REVIEW

File: docs/src/main/paradox/jdbc.md [hunk 1]
```diff
@@ -229,6 +229,14 @@ H2
 
 The schema can be created using the method `JdbcProjection.createOffsetTableIfNotExists`. This is particularly useful when writting tests. For production enviornments, we recommend creating the schema before deploying the application.
 
+@@@ warning { title=Important }
+As of version 1.0.1, the schema for PostgresSQL have changed. It now defaults to lowercase un-quoted table and column names.
```
Comment (ignasi35) [inline docs/src/main/paradox/jdbc.md:233]:
```suggestion
As of version 1.0.1, the schema for PostgreSQL has changed. It now defaults to lowercase un-quoted table and column names. 
```

File: docs/src/main/paradox/slick.md [hunk 2]
```diff
@@ -178,6 +178,14 @@ H2
 
 The schema can be created using the method `SlickProjection.createOffsetTableIfNotExists`. This is particularly useful when writting tests. For production enviornments, we recommend creating the schema before deploying the application.
 
+@@@ warning { title=Important }
+As of version 1.0.1, the schema for PostgresSQL have changed. It now defaults to lowercase un-quoted table and column names.
```
Comment (ignasi35) [inline docs/src/main/paradox/slick.md:182]:
```suggestion
As of version 1.0.1, the schema for PostgreSQL has changed. It now defaults to lowercase un-quoted table and column names. 
```

**Review by ignasi35 — APPROVED** [review]
minor typos

**Review by patriknw — APPROVED** [review]
LGTM

**Pull request merged**